modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+[id="persistent-storage-csi-migration-enable_{context}"]
+= Enabling CSI automatic migration
+.Procedure
+
+* Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
++
+The following configuration example enables CSI automatic migration to both CSI drivers (EBS and Cinder):
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
```

Comment:
How about adding a note here. If we use `TechPreviewNoUpgrade` featureset to enable csi migration, then the enable operation is undone, and will block the cluster upgrade.